src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,3 @@
+%stars-active {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
a really minor one with just single usage, but great that you put it to a practice 👍

Reply:
@meforoma I thought it would be good for the compiled version to have them all in one place